Makefile

```diff
@@ -0,0 +1,33 @@
+## help - Display help about make targets for this Makefile
+help:
+	@cat Makefile | grep '^## ' --color=never | cut -c4- | sed -e "`printf 's/ - /\t- /;'`" | column -s "`printf '\t'`" -t
+
+## install - install dependency packages
+install:
+	yarn install
+
+## dev - starts the Next.js development server on port 3000
+dev: install
+	yarn run dev
```

Comment:
Convert to `npm` please. We are not using `yarn` on this one.
`npm run dev`
Apply rest of the file